packages/blockchain-extension/extension/commands/UserInputUtil.ts

```diff
@@ -1043,23 +1043,55 @@ export class UserInputUtil {
         });
     }
 
-    public static async showNodesQuickPickBox(prompt: string, nodes: FabricNode[], canPickMany: boolean): Promise<Array<IBlockchainQuickPickItem<FabricNode>> | IBlockchainQuickPickItem<FabricNode> | undefined> {
+    public static async showNodesQuickPickBox(prompt: string, nodes: FabricNode[], canPickMany: boolean, currentNodes?: FabricNode[]): Promise<Array<IBlockchainQuickPickItem<FabricNode>> | IBlockchainQuickPickItem<FabricNode> | undefined> {
         if (nodes.length === 0) {
             throw new Error('Error when importing nodes, no nodes found to choose from.');
         }
 
         const quickPickItems: IBlockchainQuickPickItem<FabricNode>[] = UserInputUtil.selectNodesOneOrdererPerCluster(nodes);
 
+        if (currentNodes && currentNodes.length > 0) {
+            const currentUrls: string[] = currentNodes.map((_node: FabricNode) => _node.api_url);
+            quickPickItems.map((quickPickItem: IBlockchainQuickPickItem<FabricNode>) => {
+                currentNodes.forEach((node: FabricNode) => {
```

Comment:
i'm not sure a forEach is the correct thing to use here. Maybe a find to get the correct node and if it doesn't find it you know it's a new one